### ISSUE TITLE
Let admins see every report on all communities?

### ISSUE THREAD

**Issue by dessalines** [LemmyNet/lemmy#1810]
Currently, reporting is only shown to mods of communities that have a report on that community. But it would probably be beneficial for admins to see **every report** for every community, since a lot of times the main ban action should be a site ban, rather than community ban.

@Nutomic what do you think?

**Comment by Nutomic** [comment]
Yes definitely. I think the main reason for admins to exist is to oversee moderation of the instance. So they need to see everything related to that, reports in particular.
